argo-helm.tf

```diff
@@ -17,6 +40,20 @@ data "utils_deep_merge_yaml" "argo_helm_values" {
     yamlencode({
       "spec" : local.argo_application_values
     }),
+    yamlencode({
+      "spec" : {
+        "syncPolicy" : {
+          "retry" : {
+            "limit" : 5
+            "backoff" : {
+              "duration" : "30s"
+              "factor" : 2
+              "maxDuration" : "3m0s"
+            }
+          }
+        }
+      }
+    }),
```

Comment:
Is this meant to serve as a replacement for the explicit dependency between karpetner-crd and the karpetner chart? If so, can't we devise another method for this, perhaps through the available job?

Reply:
Right, can be done using K8s Job